src/components/organisms/header/header.scss

```
@@ -0,0 +1,125 @@
+@use "../../variables" as *;
+
+header {
+  background-color: $black;
+  display: inline-block;
+  height: 60px;
+  width: 100%;
+
+  @media (min-width: $breakpoint-xl) {
+    background-color: $gray_nobel;
+    display: flex;
+    justify-content: space-between;
+    height: 90px;
+  }
+
+  ul {
+    list-style: none;
+    background-color: $black;
+    color: $white;
+  }
+
+  li {
+    font-size: $fontsize-xs;
+    height: 50px;
+    line-height: 50px;
+
+    @media (min-width: $breakpoint-xl) {
+      height: 90px;
+      line-height: 90px;
+    }
+  }
+  a {
+    color: $white;
+    text-decoration: none;
+    font-size: $fontsize-s;
+
+    &:hover {
+      color: $white;
+    }
+  }
+}
+
+.header {
+  display: flex;
+}
+
+.header-img {
+  height: 20px;
+  margin: 20px 16px;
+
+  @media (min-width: $breakpoint-xl) {
+    height: 24px;
+    margin: 33px 30px;
+  }
+}
+
+.humberger-btn {
+  margin-left: auto;
+}
+
+#humberger {
+  img {
+    height: 15px;
+    margin: 22.5px 16px;
+  }
+}
+
+.concept {
+  display: flex;
+
+  img {
+    width: 26px;
+    height: 26px;
+    margin: 12px 16px;
+  }
+}
+
+.card {
+  border: none;
+}
+
+#main-ul {
+  @media (min-width: $breakpoint-xl) {
+    background-color: $gray_nobel;
+    display: flex;
+    margin: 0px;
+    padding: 0px;
+  }
+}
+
+.main-li {
+  @media (min-width: $breakpoint-xl) {
+    margin-right: 30px;
+    transition: transform 0.2s ease;

```

```suggestion
    transition:  0.2s ease;

```
transformを付けると`transform`プロパティ（要素の拡大、回転などに使用されます）に発生する変化に対してスタイルを指定することになる（transformを付けると次のようなスタイルが適用されます。` transition-property: transform;`）ので今回のようにホバー時の色変化だけの場合指定しないほう（つけないと次のようにスタイルが適用されます。` transition-property: all;`）がうまくいきます。